benchmarks/dynamo/operatorbench/utils/common.py

```diff
@@ -0,0 +1,50 @@
+import dataclasses
+import pathlib
+from enum import Enum
+from typing import Dict, List, Optional
+
+import torch
+
+from .metrics import Device, Metrics
+
+
+@dataclasses.dataclass
+class OperatorConfig:
+    name: str
+    variant: str
+    device: str
+    extra_args: List[str]
+    extra_env: Optional[Dict[str, str]] = None
+    output_dir: Optional[pathlib.Path] = None
+
+
+class BenchmarkResults:
+    def __init__(self, durations: List[float]):
+        self.durations = durations
+
+    def median(self) -> float:
+        # return np.median(self.durations)
+        pass
+
+
+@dataclasses.dataclass
+class BenchmarkConfig:
+    device: Device
+    dtype: torch.dtype
+    phase: str
+    max_samples: int
+    repeat: int
+    metrics: List[Metrics]
+
+
+class Phase(Enum):
+    FORWARD = "forward"
+    BACKWARD = "backward"
+    FULL = "full"
```

Comment:
Add unit test to exercise each of these phases for a example operator

Reply:
will do in next commit

Reply:
do you have any suggestions on where I should add a unit test?

Reply:
I don't have much preference. Since inductor tests are under test/inductor/ . For operatorbench, you may add tests under test/operatorbench. Or if you only expect a single file, just add test/test_operatorbench.py

Reply:
added in https://github.com/pytorch/pytorch/pull/136169/commits/2d93c5f720e2fbd0e3e6d3c9086de9d35bcb18e0